draft-ietf-tls-tls13.md

```diff
@@ -1615,7 +1620,7 @@ Structure of this message:
            ProtocolVersion version;
            Random random;
            CipherSuite cipher_suite;
-           Extension extensions<0..2^16-1>;
+           Extension extensions<1..2^16-1>;
```

Comment:
The server's pre_shared_key is a uint16 selected_identity, which would make it 6.
(KeyShareEntry has a NamedGroup and a non-empty key_exchange).